setup/stability-extra/multicluster-vpn/default/setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo
+echo " [*] Installing the 'default' scenario"
+echo
+
+SCENARIO="default"
+NAMESPACE="env-$SCENARIO"
+$KUBECTL2 create ns $NAMESPACE || true
+
+$KUBECTL2 -n $NAMESPACE delete secrets -l "istio/multiCluster=true"
+install_k8s_secrets "$KUBECONFIG2" "$KUBECONTEXT2" "$NAMESPACE" "$KUBECONFIG1" "$KUBECONTEXT1" "istio-system"
+
+cat > ${TEMP_DIR}/values-${SCENARIO}.yaml <<EOF
+global:
+  hub: ${HUB}
+  tag: ${TAG}
+  istioNamespace: ${NAMESPACE}
+  configNamespace: ${NAMESPACE}
+  telemetryNamespace: ${NAMESPACE}
+  policyNamespace: ${NAMESPACE}
+  configValidation: false
+  proxy:
+    tracer: ""
+pilot:
+  configNamespace: ${NAMESPACE}
+EOF
+
+helm template \
+  --namespace $NAMESPACE \
+  -n galley-$NAMESPACE \
+  $ISTIO_INSTALLER_PATH/istio-control/istio-config/ \
+  -f $ISTIO_INSTALLER_PATH/global.yaml \
+  -f ${TEMP_DIR}/values-${SCENARIO}.yaml \
+  | $KUBECTL2 apply -f -
+
+helm template \
+  --namespace $NAMESPACE \
+  -n pilot-$NAMESPACE \
+  $ISTIO_INSTALLER_PATH/istio-control/istio-discovery/ \
+  -f $ISTIO_INSTALLER_PATH/global.yaml \
+  -f ${TEMP_DIR}/values-${SCENARIO}.yaml \
+  | $KUBECTL2 apply -f -
+
+helm template \
```

Comment:
I don't think we need this if we are using manual injection?

Reply:
Even for manual injection we need an injector configmap. I found it more straightforward to simply install autoinjector and point istioctl to that NS. Also I would be cool to get the autoinjection working at some point as well.

Reply:
Please don't - we have a test that is using manual inject without depending on k8s. 

People look at the tests as examples - we don't want to require access to k8s to do manual injection, it was a mistake.